.travis.yml

```diff
@@ -14,4 +14,5 @@ go_import_path: github.com/lionelbarrow/braintree-go
 script:
   - source .default.env
   - go vet ./...
+  - CGO_ENABLED=0 errcheck ./...
```

Comment:
`CGO_ENABLED=0` is required to prevent errcheck from utilizing gcc. Apparently this is a bug in the latest version of Go, since `CGO_ENABLED` is supposed to default to `0` but it's defaulting to `1`.